docs/pages/careers/code-infra-engineer.md

```diff
@@ -0,0 +1,139 @@
+# Code infra engineer
+
+<p class="description">You will drive the development and maintenance of the infrastructure that powers all MUI products.</p>
+
+## Details of the role
+
+- **Location**: Remote (preference for UTC-6 to UTC+5).
+- **Type of work**: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
+- **Level**: [IC4 or above](https://mui-org.notion.site/Leveling-at-MUI-5c30f9bfe65149d697f346447cef9db1).
+- We're a **remote** company, operating mostly asynchronously.
+
+## The company
+
+MUI's story began in 2014 with Material UI, the most successful React implementation of Google's Material Design.
+Today, Material UI stands as one of the most popular open-source libraries on GitHub and has paved the way for the fully-fledged startup known as MUI (founded in 2019), which now boasts an ever-expanding ecosystem of React UI products.
+We're a company of 35+ people as of early 2024, and we're growing.
+
+## The products
+
+MUI is best known for our flagship product, Material UI—but this is just one of three core component libraries we maintain.
+Base UI is our headless component library, and Pigment CSS is a zero-runtime CSS-in-JS styling engine.
+We also host Design Kits and pre-built Templates.
+
+Beyond the core libraries, MUI X offers advanced components like the Data Grid, Date and Time Pickers, and Charts, for more complex user interactions and data visualization needs.
+
+We're also making ambitious moves to incorporate our full suite of components into Toolpad, a low-code admin builder tool for assembling full-stack apps faster than ever.
+
+Learn more about MUI's products in this blog post: [An introduction to the MUI ecosystem](https://mui.com/blog/mui-product-comparison/).
+
+## The culture
+
+MUI is a fully remote company with a team that spans the globe.
```

Comment:
```suggestion
MUI is a fully remote company with a team that spans across the globe.
```

Reply:
Or
```suggestion
MUI is a fully remote company with a team that spans the entire globe.
```
?

Reply:
It's not really entire :D I would use across, we may want to update the template with this change later, I will handle it separately.